Final

anmelll opened the pull request.

anmelll committed:
Final project DNA

slebedev commented on DNA/DNA/main.m:
```diff
@@ -0,0 +1,137 @@
+// реализация метода замены 
+-(void) mutator:(int)x
+{
+    // объявление массива для контроля использаванных индексов
```
х - процентный показатель, и в задании это сказано. Учтите это в реализации метода mutate.

anmelll committed:
Версия 2, дополненная

Добавлена проверка на %

anmelll committed:
Версия 2

Добавлена проверка на %

anmelll committed:
Версия 2

Добавлена проверка на %

anmelll committed:
Final Версия 3

Перенес проверку на % в метод mutator. Поправил реализацию ввода %
(пока не будет введено значение из необходимого диапазона)

anmelll committed:
Final Версия 3.1

Перенес проверку на % в метод mutator. Поправил реализацию ввода %
(пока не будет введено значение из необходимого диапазона)

anmelll commented:
Проверку перенес в метод. Значение теперь будет вводится пока пользователь не введет число в нужном диапазоне

anmelll committed:
Revert "Final Версия 3.1"

This reverts commit eb85690e49b74f74eb2c84972986394283ccc478.

anmelll committed:
Project DNA Final 3.2
